packages/botbuilder-adapter-hangouts/package.json

```diff
@@ -35,16 +35,19 @@
   },
   "dependencies": {
     "botbuilder": "^4.10.1",
-    "googleapis": "^34.0.0",
     "botkit": "^4.10.0",
-    "debug": "^4.1.0"
+    "cross-fetch": "^3.0.6",
```

Comment:
I used cross-fetch here since I was familiar with it. If there's another preferred way to make api calls, I'm more than happy to make any required changes.